tests/Feature/Modules/Crm/TitleCrudTest.php

```
@@ -0,0 +1,117 @@
+/**
+ * @group smoke
+ */
+class TitleCrudTest extends TestCase
+{
+    protected User $user;
+
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        Model::unguard();
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Database\Eloquent\Model' in method 'setUp'. (undefined)

(StaticAccess)